products/memgraph/upcoming/quick-start.md

```diff
@@ -214,25 +214,29 @@ The easiest way to execute openCypher queries against Memgraph is by using
 Memgraph's command-line tool, `mg_client`, which is installed
 together with Memgraph.
 
+NOTE: `mg_client` is deprecated tool still coming within Memgraph package.
```

Comment:
```suggestion
---
**NOTE**

`mg_client` is deprecated tool still coming within Memgraph package.
[mgconsole](https://github.com/memgraph/mgconsole) will replace `mg_client` in
the future. If possible, please use `mgconsole` instead.

---
```

Reply:
Because of GitBook https://docs.gitbook.com/editing-content/markdown#hints-and-callouts?